framework/packages/abstract-interface/src/account/mod.rs

```diff
@@ -190,6 +190,52 @@ impl<Chain: CwEnv> AbstractAccount<Chain> {
     {
         self.manager.register_remote_account(host_chain)
     }
+
+    pub fn create_sub_account(
+        &self,
+        account_details: AccountDetails,
+        funds: Option<&[Coin]>,
+    ) -> Result<AbstractAccount<Chain>, crate::AbstractInterfaceError> {
+        let AccountDetails {
+            name,
+            description,
+            link,
+            namespace,
+            base_asset,
+            install_modules,
+        } = account_details;
+
+        let result = self.manager.execute(
+            &abstract_core::manager::ExecuteMsg::CreateSubAccount {
+                name,
+                description,
+                link,
+                base_asset,
+                namespace,
+                install_modules,
+            },
+            funds,
+        )?;
+
+        // Parse data from events
+        let acc_seq = &result.event_attr_value(ABSTRACT_EVENT_TYPE, "account_sequence")?;
```

Comment:
I think this functionality is implemented in some other places as well right? Might be worth pulling it into its own fn

Reply:
Pulled it into own fn